src/current/v23.2/cockroach-debug-merge-logs.md

```diff
@@ -24,10 +24,11 @@ $ cockroach debug merge-logs [log file directory] [flags]
 
 ## Flags
 
-Use the following flags to filter the `debug merge-logs` results for a specified regular expression or time range.
+The `debug merge-logs` subcommand supports the following flags:
 
 Flag | Description
 -----|-----------
+`--format` | **Required** The [format of the logs]({% link {{ page.version.version }}/log-formats.md %}) in the log file directory. The logs in a `.zip` generated using [`cockroach debug zip`]({% link {{ page.version.version }}/cockroach-debug-zip.md %}) are of type [`crdb-v1`]({% link {{ page.version.version }}/log-formats.md %}#format-crdb-v1).
```

Comment:
This tells me what is going to be in v1.  How do I know what logs are going to be in the v2 format?  

I guess as a user I should know whether my logs are v1 or v2 because I specified it in the [YAML logging config](https://www.cockroachlabs.com/docs/v23.2/configure-logs#yaml-payload) at start time right?  Sorry if these are dumb questions, just trying to make sure I understand

Reply:
To clarify,
- If you run `debug zip merge-logs` against the actual logs of a cockroach node you will use the format that you specified for the logs.
- If you run `debug zip merge-logs` against the logs unzipped from a generated `.zip` file, the format will be `crdb-v1` because `debug zip` converts the whatever the format of the actual logs into `crdb-v1`.
Does that make sense? We were surprised that this was the case.